<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>816</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>等线</family>
    <pointsize>11</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="open_serial_btn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QComboBox" name="serial_port_cb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>87</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="serial_baudrate_txt">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
    <property name="placeholderText">
     <string>BaudRate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_serial_btn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>1171</width>
      <height>761</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="variable_tab">
     <attribute name="title">
      <string>Variable Tab</string>
     </attribute>
     <widget class="QCustomPlot" name="custom_plot" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1141</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="debug_info_txt">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>801</width>
        <height>301</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Printf Information</string>
      </property>
     </widget>
     <widget class="QTableView" name="variable_list">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>420</y>
        <width>331</width>
        <height>301</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>80</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="status_tab">
     <attribute name="title">
      <string>Status Tab</string>
     </attribute>
     <widget class="Compass" name="compass_widget" native="true">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>40</y>
        <width>301</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Compass</class>
   <extends>QWidget</extends>
   <header>compass.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
